src/coreclr/jit/gentree.cpp

```diff
@@ -2519,6 +2519,12 @@ bool GenTreeCall::Equals(GenTreeCall* c1, GenTreeCall* c2)
         return false;
     }
 
+    // Make sure "explicit tail call" flags match.
+    if (c1->IsTailPrefixedCall() != c2->IsTailPrefixedCall())
```

Comment:
I think this should be reverted (and fixed in the future PR), or we should check all the tailcall flags.

Reply:
Reverted